### PR TITLE
Playlist

### LINKED ISSUE #5: My playlist layout

### PR DESCRIPTION
**Pull request by frederiktt**

### CODE REVIEW

**Review by JakobKaae — APPROVED** [review]
Looks good

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,8 +9,8 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".HomeActivity"
-            android:label="@string/app_name">
```
Comment (kvinkel) [inline app/src/main/AndroidManifest.xml]: Label got removed

**Review by JakobKaae — APPROVED** [review]
Naicu

**Review by kvinkel — APPROVED** [review]
Looks good! :thumbsup:

**Pull request merged**